tumblr-photo-video-ripper.py

```diff
@@ -82,15 +88,16 @@ def _register_regex_match_rules(self):
     def _handle_medium_url(self, medium_type, post):
         try:
             if medium_type == "photo":
-                return post["photo-url"][0]["#text"]
+                return post["url"]
```

Comment:
I didn't see any handlings for `post["alt_sizes"]`, which would contain photos with different resolutions. The largest one should be picked up.